### PR TITLE
Align with arduino serever

### PR DESCRIPTION
**Pull request by YigalB1**

### CODE REVIEW

File: Car_motor_nano_Shield_inwork.ino [hunk 1]
```diff
@@ -45,8 +47,11 @@ void setup() {
   pinMode(trigPin, OUTPUT); 
   pinMode(echoPin, INPUT); 
   
-  myservo.attach(7);  // attaches the servo on pin 7 to the servo object
-  
+  #if 1==NANO
+    myservo.attach(7);  // Use pin 7 in case of NANO
+  #else
+    myservo.attach(5);  // Use pin 5 in case of UNO
```
Comment (adi-lb) [inline Car_motor_nano_Shield_inwork.ino]: in the code here - this line seems like a comment. not sure if it is right or what it means...
please take a look and see if there is anything to do here or it is OK

File: Car_motor_nano_Shield_inwork.ino [hunk 2]
```diff
@@ -25,10 +26,11 @@ const int GO        = 3;
 const int LEFT      = 4;
 const int RIGHT     = 5;
 const int trigPin   = 2;  // Ultrasonic (Yellow)
-const int echoPin   = 3;  // Ultrasonic (orange)
+const int echoPin   = 4;  // Ultrasonic (orange)
 
 int dist_array[SERVO_STEPS_NUM];
-// TBD add one to the size of array
```
[the remark this answers is not in the document]
Comment (YigalB1) [reply Car_motor_nano_Shield_inwork.ino]: agree

[the remark this answers is not in the document]
Comment (YigalB1) [reply Car_motor_nano_Shield_inwork.ino]: How do I do that?
Change only this line?

[the remark this answers is not in the document]
Comment (YigalB1) [reply Car_motor_nano_Shield_inwork.ino]: Do I need to create a new branch just for that?

File: Car_motor_nano_Shield_inwork.ino [hunk 3]
```diff
@@ -324,7 +320,7 @@ int readDistance() {
     // Serial.print(duration_t);
     //Serial.print("dist_t= ");
     // Serial.println(dist_t);
-  #endif 
+  //#endif 
   
   return(dist_t);
-}
+}
```
[the remark this answers is not in the document]
Comment (YigalB1) [reply Car_motor_nano_Shield_inwork.ino]: ignore

**Pull request merged**